Improve chinese_simplified localization

violarulan opened the pull request:
In the Chinese localization file, there are some missing translations, resulting in several English words appearing.

This PR brings the localization to where it should be and standardizes existing translation phrases.

HB9HIL commented on application/language/chinese_simplified/filter_lang.php:
```diff
@@ -105,21 +105,21 @@
 $lang['filter_actions_print_include_via'] = "包含通过";
 $lang['filter_actions_print_include_grid'] = '包含网格？';
-$lang['filter_actions_print_include_ref'] = "Include reference? (SIG, SOTA, POTA, IOTA, WWFF; If available in location)";
+$lang['filter_actions_print_include_ref'] = "Include reference?(SIG, SOTA, POTA, IOTA, WWFF; If available in location)";
```
Can you translate this line aswell? :-)

HB9HIL commented:
Hello @violarulan 

Thank you very much for this improvement :)

Can you translate the marked line aswell to make it complete?

HB9HIL commented:
Additionally I noticed you based your PR against `master` branch. I switched it over to `dev` branch. So you can see there would be much more

violarulan commented:
> Additionally I noticed you based your PR against `master` branch. I switched it over to `dev` branch. So you can see there would be much more
> 
> perform `git pull` before adding a new commit

Sure thing and thanks for the correction. I've added more translation. It should be completed by now.

_...and please ignore my misuse of issue referencing in commit message._

HB9HIL reviewed (approved):
Thank you very much! Contributing Language is really important. We appreciate it :)

HB9HIL commented:
Hello @violarulan 

We switched over to `gettext()` and po files. Now much more chinese translations are waiting for translations :)

Check this here: https://translate.wavelog.org/engage/wavelog/

violarulan commented:
Thanks @HB9HIL

That's a good translation platform. I've registered an account. I'll do the translation work there.
